include/dali/core/boundary.h

```diff
@@ -59,6 +62,45 @@ struct Boundary {
   T value = {};
 };
 
+inline std::string to_string(const BoundaryType& type) {
+  switch (type) {
+    case BoundaryType::CONSTANT:
+      return "constant";
+    case BoundaryType::CLAMP:
+      return "clamp";
+    case BoundaryType::REFLECT_1001:
+      return "reflect_1001";
+    case BoundaryType::REFLECT_101:
+      return "reflect_101";
+    case BoundaryType::WRAP:
+      return "wrap";
+    case BoundaryType::TRANSPARENT:
+      return "transparent";
+    case BoundaryType::ISOLATED:
```

Comment:
This is just a flag that can be combined with other border modes.

Reply:
Good point, it is no longer used by the kernel, the "valid" mode will be part of the operator, at the kernel level it can be just achieved with the implicit roi handling (output smaller than the input + anchor).